snuba/web/rpc/__init__.py

```diff
@@ -101,5 +112,44 @@ def _after_execute(
 }
 
 
-for version, module_path in _TO_IMPORT.items():
-    import_submodules_in_directory(module_path, f"snuba.web.rpc.{version}")
+for v, module_path in _TO_IMPORT.items():
+    import_submodules_in_directory(module_path, f"snuba.web.rpc.{v}")
```

Comment:
why the change lol

Reply:
mypy fails because "version" is in the global scope here, is easier to just rename

Reply:
for some reason python hoists loop variables into the global scope, annoying